js/sign.js

```diff
@@ -0,0 +1,79 @@
+const signInput = document.querySelector('.sign-input');
+const inputError = document.querySelector('.input-error');
+const inputPassword = document.querySelector('#password-input');
+const inputErrorPassword = document.querySelector('.input-error-pwd');
+const signForm = document.querySelector('.sign-form');
+
+const isEmpty = (e, inputValue, inputError) => {
+  // input이 비어있는경우
+  if (!inputValue) {
+    const { title } = e.target.dataset;
+    inputError.textContent = `${title}을 입력해주세요.`;
+    e.target.classList.add('error');
+  }
+};
+
+const validateEmail = (e, inputValue, inputError) => {
+  // 이메일 형식이 맞지 않는 경우
+  const re = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  if (!re.test(inputValue)) {
+    inputError.textContent = '올바른 이메일 주소가 아닙니다.';
+    e.target.classList.add('error');
+  }
```

Comment:
```suggestion
  if (re.test(inputValue)) {
    return;
  }
  
   inputError.textContent = '올바른 이메일 주소가 아닙니다.';
   e.target.classList.add('error');
```
이런식으로도 구현할 수 있을 것 같아요!